### PR TITLE
UHM-4565 Order Labs page on tablet

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-4565

Sorry for the huge refactor blended in with formatting change.

I have verified that the following still works:
- List of lab orders page:
  - paging the order list
  - Add Lab Order button
- New lab order page:
  - Changing set
  - Selecting panels and tests
  - Viewing panel tooltip
  - Mark as urgent
  - Remove panel or test from draft orders
  - Return button
  - Discard All button
  - Sign and Save button

There should be no functional change aside from the formatting changes.

Almost all of the Javascript changes were just unused code, not referenced or used anywhere. In some cases, components that did almost nothing were un-abstracted into their unique parent component.

Lots of CSS was deleted. Most of it was just not referenced anywhere, but in many cases it was no longer necessary, or never was.

Screenshot from 2020-05-28 15-50-18
Screenshot from 2020-05-28 15-50-28
Screenshot from 2020-05-28 15-51-55
Screenshot from 2020-05-28 15-54-51
Screenshot from 2020-05-28 15-52-47

### CODE REVIEW

File: app/css/openmrs-owa-orderentry.scss [hunk 1]
```diff
@@ -34,41 +34,37 @@ body {
...
-  height: 100%;
-  padding-top: 12px;
-  padding-right: 5px;
+  @media (min-width: 992px) {
+    .col-md-#{$i}.col-md-#{$i}.col-md-#{$i} {  // specificity x3
+      grid-column: span $i;
+    }
+  }
 }
+/* End layout */
```
Comment (brandones) [inline app/css/openmrs-owa-orderentry.scss:67]: Very easy way to imitate Bootstrap layout behavior.

File: app/js/components/labOrderEntry/LabEntryForm.jsx [hunk 2]
```diff
@@ -168,48 +147,70 @@ export class LabEntryForm extends PureComponent {
...
+                </form>
               </div>
-              <button className="cancel" disabled={!!allDraftOrders} onClick={() => window.location.assign(backLink)}>Return</button>
             </div>
-            :
-            <p>No Lab Orderables was found</p>
-        }
+          </div>
+        ) : (
+          <p>No Lab Orderables was found</p>
```
Comment (mogoodrich) [inline app/js/components/labOrderEntry/LabEntryForm.jsx:212]: Oops... looks like some unlocalized text we missed back in the day.

File: app/js/components/labOrderEntry/LabEntryForm.jsx [hunk 3]
```diff
@@ -140,25 +138,6 @@ export class LabEntryForm extends PureComponent {
...
-      />
-      <LabTestFieldSet
-        labCategoryName={this.state.categoryName}
-        handleTestSelection={this.handleTestSelection}
-        selectedTests={this.props.selectedTests}
-        tests={this.props.standAloneTests}
-        locale={this.props.sessionReducer.locale}
-      />
-    </div>
-  );
```
Comment (mogoodrich) [inline app/js/components/labOrderEntry/LabEntryForm.jsx:160]: Yeah, it's easier to read without having these function blocks that just render and aren't reused. 

Are there any other advantages or disadvantages of making this move?

Comment (mogoodrich) [reply app/js/components/labOrderEntry/LabEntryForm.jsx:160]: (Oh, yeah, I see you referenced this in your overall comments... sounds good to me)

Comment (brandones) [reply app/js/components/labOrderEntry/LabEntryForm.jsx:160]: No disadvantages I can think of. I think React best practice is that all the JSX lives in a React component's render method (or the return statement of a React function component).

**Pull request merged**